app/controllers/api/v1/posts_controller.rb

```diff
@@ -4,6 +4,10 @@ def image_upload
     render json: { filelink: '/uploads/post/thumb/1/1203251823070dd2468eaff085.jpg' }
   end
 
+  def welcome
+  	render text: 'welcome two'
```

Comment:
Use 2 (not 1) spaces for indentation.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.<br>Tab detected.